client/src/components/TrackList.vue

```diff
@@ -135,6 +136,27 @@ export default defineComponent({
     watch(selectedTrackIdRef, scrollToTrack);
     watch(tracksRef, scrollToSelectedTrack);
 
+
+    async function multiDelete() {
+      const tracksDisplayed: number[] = [];
+      const text = ['Do you want to delete the following tracks:'];
+      virtualListItems.value.forEach((item) => {
+        if (item.checkedTrackIds.includes(item.track.trackId)) {
+          tracksDisplayed.push(item.track.trackId);
+          text.push(item.track.trackId.toString());
```

Comment:
This will get a bit long, won't it?  Like if I just want to clear out the tracks, this could be tens of thousands of lines that the prompt has to render.

Reply:
Yeah I was wondering if I should truncate it after like 100 items if it is longer.